### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by JoeZZG**
Added introduction about the COBOL programming language with some tutorial on basic operations of COBOL, including math operations, function call and some example to use the language.

### CODE REVIEW

File: Topics/Tech_Stacks/COBOL/Basic_Syntax.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+000600 01 NUM-B PIC 9(4) VALUE 5678.
+000700 01 RESULT PIC 9(5).
+000800
+000900 PROCEDURE DIVISION.
+001000    COMPUTE RESULT = NUM-A + NUM-B.
+001100    DISPLAY "The result of adding " NUM-A " and " NUM-B " is: " RESULT.
+001200 STOP RUN.
+```
+
+## Understanding the Initial Lines of a COBOL Program
```
Comment (tlan4) [inline Topics/Tech_Stacks/COBOL/Basic_Syntax.md:90]: I think moving this section to the start would be better, as it helps readers understand the previous examples better

File: Topics/Tech_Stacks/COBOL/Basic_Syntax.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+## Variable Declaration
+
+In COBOL, variables are declared in the `DATA DIVISION` section of the program.
+A typical declaration specifies the variable's level number, name, and type.
+
+```cobol
+DATA DIVISION.
+WORKING-STORAGE SECTION.
+01 NUM-A PIC 9(3).     -- Declares an integer variable with 3 digits
```
[the remark this answers is not in the document]
Comment (JoeZZG) [reply Topics/Tech_Stacks/COBOL/Basic_Syntax.md]: Thank you for your advice! I have already committed my work with extra explanation on variable assignment!

File: Topics/Tech_Stacks/COBOL/Basic_Syntax.md (hunk 1 above)
Comment (JoeZZG) [reply Topics/Tech_Stacks/COBOL/Basic_Syntax.md:95]: In my opinion, it is better to let the student learn some basic syntax then they can learn how to implement some codes.

File: Topics/Tech_Stacks/COBOL/Basic_Syntax.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+COMPUTE RESULT = NUM-A * NUM-B.    -- Multiplication
+COMPUTE RESULT = NUM-A / NUM-B.    -- Division
+```
+
+## Function Calls
+In COBOL, a function call is made using the CALL statement,
+where you specify the function name and pass any required parameters.
+
+```cobol
+CALL 'FUNCTION-NAME' USING PARAM1 PARAM2.
```
[the remark this answers is not in the document]
Comment (JoeZZG) [reply Topics/Tech_Stacks/COBOL/Basic_Syntax.md:37]: No problem, I will give more specific examples on how to define a function and how each step work!

**Comment by tlan4** [comment]
I think your changes look good, much better explanation for a beginner

**Pull request merged**